libcodechecker/analyze.py

```diff
@@ -238,6 +238,10 @@ def main(args):
     # Parse the JSON CCDBs and retrieve the compile commands.
     actions = []
 
+    if len(args.logfile) != 1:
```

Comment:
`logfile` flag is declared to be a string in the command line arguments. Isn't it going to check the length of that string? This can be an array if this `analyze` module is loaded from a wrapper module (e.g. `CodeChecker check`) which gives it an array. By the way `CodeChecker check` gives it an array of length 1. Can this array be longer?